### DISCUSSION TITLE
Server Error 400 Creating Relationship Column

### DISCUSSION THREAD

**Discussion by odysseyskate** [Budibase/budibase#3154] [Help]
I'm attempting to add a new table column that is a relationship type. No matter what options I choose (trying to link any column & any relationship type) I get error 400 through the debug menu in Chrome. The GUI just goes back to the main page without showing an error.

Please help!

**Comment by mjashanks** [comment]
Hi there, please paste the console error in here.

Also, it would be useful to see any failed network requests, with their associated response.

Thanks!

**Reply by odysseyskate** [reply]
"Failed to load resource: the server responded with a status of 400 ()"

Network console says ""Cannot overwrite existing column.""

So there's the problem ^

Turns out to be user error after all - I didn't expect the "Column name in other table" to be a new column name - I was trying to link it to an existing. Not an issue though as it's a new project anyway.

Thanks for your help!
